cupy/cuda/cufft.pyx

```diff
@@ -713,23 +722,28 @@ class PlanNd(object):
             result = cufftCreate(&plan)
             if result == 0:
                 result = cufftSetAutoAllocation(plan, 0)
-            if result == 0:
-                result = cufftMakePlanMany(plan, ndim, shape_ptr,
-                                           inembed_ptr, istride, idist,
-                                           onembed_ptr, ostride, odist,
-                                           <Type>fft_type, batch,
-                                           &work_size)
+        check_result(result)
```

Comment:
You should hold the plan immediately after it's created. Remember to remove Line 758 below that does the same thing.
```suggestion
        check_result(result)
        self.plan = plan
```

Reply:
Thanks, I fixed!